Source/WebCore/css/mediaControls.css

```diff
@@ -262,7 +258,7 @@ video::-webkit-media-text-track-region-container {
     flex-direction: column;
 }
 
-video::-webkit-media-text-track-region-container.scrolling {
+video[pseudo="-webkit-media-text-track-region-container.scrolling"].scrolling {
```

Comment:
```suggestion
video[pseudo="-webkit-media-text-track-region-container"].scrolling {
```

Reply:
Done in [3fb18c8](https://github.com/WebKit/WebKit/commit/3fb18c8f7d1167791a66c1904ffbf1a107168d4d).